### PR TITLE
fix: servicemonitor labels

### LINKED ISSUE #626: The `NGINX Ingress controller` dashboard in Grafana does not show anything.
**Describe the bug:** (a clear and concise description of what the bug is)

The `NGINX Ingress controller` dashboard in Grafana does not show anything.  
**To Reproduce**

1. Open Otomi Console in webbrowser
2. Go to OtomiApps -> Grafana -> Dashboards -> Manage
3. From Dashboard list select `NGINX Ingress controller`
4. Observe that there are not any metrics displayed 

**Expected behavior:** (a clear and concise description of what you expected to happen)
See screenshots
**Screenshots:** (if applicable, add screenshots to help explain your problem)
Current

Expected:

**Cluster(s):**

- Cloud: [e.g. aws]
- Cluster: [e.g. demo]

**Versions:** (seen in console's `/settings`)

- Core Version 0.14.17

### LINKED ISSUE #820: most servicemonitors are not working
**Describe the bug:** (a clear and concise description of what the bug is)
Due to prometheus instance losing label config due to regression, a lot of servicemonitors are not connected to their prometheus instance.

### PR DESCRIPTION
**Pull request by Morriz**
fixes #626
fixes #820

### CODE REVIEW

**Comment by Morriz** [comment]
Not an interesting PR to review imo, but important to release soon as half of otomi workloads are now not tracked!

**Review by j-zimnowoda — COMMENTED** [review]
I am review it

File: values/prometheus-operator/service-monitors.yaml [hunk 1]
```diff
@@ -1,69 +1,70 @@
...
+    selector:
+      matchLabels:
+        istio: ingressgateway-public
+    namespaceSelector:
+      matchNames:
+        - istio-system
+    endpoints:
+      - targetPort: http-envoy-prom
+        path: /stats/prometheus
+  - name: istio-ingressgateway-private
```
Comment (j-zimnowoda) [inline values/prometheus-operator/service-monitors.yaml:24]: the ingressgateway-private is deprecated, then perhaps it does not make sense to collect metrics from it

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Hi , while testing I noticed that the Grafana dashboard for NGINX Ingress controller, still does not show metrics.

image

**Review by j-zimnowoda — COMMENTED** [review]
Besides that it looks good. I have checked if metrics are visible in Istio Workload and Istio Service dashboard and saw logs for teams and platform service.

If you just remove reference to fixes https://github.com/redkubes/otomi-core/issues/626 then we can merge it

File: values/prometheus-operator/service-monitors.yaml (hunk 1 above)
Comment (Morriz) [reply values/prometheus-operator/service-monitors.yaml:24]: removed

**Pull request merged**